flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
this is disappointing.

Reply:
also it should be `only-arches` instead. because that's basically what you limit it to.

Reply:
Fixed